docs/user-guide/102-configuration.md

```diff
@@ -176,8 +175,9 @@ status:
 
 The output contains the `desiredState` applied by the Policy for the given Node.
 It also contains a list of conditions. This list is more detailed than the one
-in Policy. It shows whether the Policy matched given Node
-(`AllSelectorsMatching`), if the `desiredState` is currently being applied on
+in Policy. It shows whether the max unavailable node limits has reached so the
+node has to wait for the networking node to be applied ('Pending'),
```

Comment:
we should use return quotes here: ('Pending') -> (\`Pending\`)

Reply:
Done